apps/prairielearn/src/ee/webhooks/stripe/index.ts

```diff
@@ -99,6 +99,14 @@ router.post(
     ) {
       const session = event.data.object as Stripe.Checkout.Session;
       await handleSessionUpdate(session);
+    } else if (event.type === 'product.updated') {
+      const product = event.data.object as Stripe.Product;
+      const productId = product.id;
+      await clearStripeProductCache(productId);
+    } else if (event.type === 'price.updated') {
+      const price = event.data.object as Stripe.Price;
+      const productId = typeof price.product === 'string' ? price.product : price.product.id;
```

Comment:
I thought the cached product only has a default price ID in it, not the actual price itself, so we wouldn't need to clear the product cache in this case?